docs/CONTRIBUTING.md

```diff
@@ -208,7 +208,22 @@ To run tests locally before pushing your code, refer to [TESTING] guide.
 
 ## Document
 
-PR should include changes in relevent documentation along with code changes.
+- PR should include changes in relevant documentation along with code changes.  PR is checked by bot to have either 
+of the following
+  - A commit that follows [commit guidelines](#commits) with `docs:` message 
+  - Changes in artifacts under `jans/docs`
+  
+- If PR does not need any documentation changes, then developer need to acknowledge that too in one of two ways:
```

Comment:
If PR does not need any documentation changes, then **the** developer need**s** to acknowledge that ~~too~~ in one of two ways:

Reply:
Fixed with [cf97cff](https://github.com/JanssenProject/jans/pull/1797/commits/cf97cffb27c8e76fe559192c3456f372f6eaa20d)